### DISCUSSION TITLE
Sostituzione centralina allarme filare, lista della spesa

### DISCUSSION THREAD

**Discussion by jucasoft** [indomus/forum#480] [Hardware & Firmware]
Ciao a tutti, preso da un momento di delirio ho strappato dal muro la centralina del mio allarme domestico.

ho recuperato la scatola metallica, ho dissaldato tutti gli zoccolini a vite (volevo vedere se ero ancora capace ad usare il succhiastagno) e tutto il reso l'ho buttato in discarica.

ora mi serve una nuova centralina.

ho cercato dei moduli di espansione per raspberry, quello che mi fa più sangue è [modulo di espansione sexy](https://www.reichelt.com/it/it/-rpi-unipi-1-1-p253692.html?PROVID=2814&gclid=CjwKCAjwoNuGBhA8EiwAFxomA-UMHVvBEk6F45x3qUnv1mdMTOkZB4G24-mmw1zjfI8f140NkYaiIRoCCSMQAvD_BwE)

non vorrei comprarlo per poi scoprire che non si tratta della scelta migliore.

vi chiedo supporto nel comprare:
 - il necessario per collegarmi al sistema di allarme filare
    - consigliate una scheda di un allarme che poi si interfacci con Raspberry o un modulo di espansione per raspberry?
 - moduli per inviare sms/3g/wifi
 - eventuali moduli per interfacciarsi con protocolli per l'automazione.
 - 

spero nel vostro aiuto.
ah! non vorrei spendere più di 500 euro.

**Comment by Macgiver1976** [comment]
Ciao, sto facendo la stessa cosa! 
Non conosco quell'espansione, ho fatto in modo da eliminarla e portato tutti i contatti nel box (Paradox) dove ho installato il Raspberry.
Ho aggiunto dei contatti MII alle finestre tramite il protocollo ZigBee radio. 
Per il combinatore GSM ti posso consigliare l'AMC VoxOut, io l'ho collegato a dei relè e funziona alla grande. 
Se non lo hai già fatto, prevedi anche il funzionamento del sistema con alimentazione assente. Io ho usato un alimentatore di una centrale Bentel a cui ho collegato una batteria 12V 2 Ah come tampone e un riduttore da 12 a 5V - 10 Ah per alimentare il tutto, sensori compresi, in più ho collegato un relè alla 12V dell'alimentatore in modo che mi avvisa se manca la corrente 220V tramite un automazione di HA. 
Sono fermo perchè non trovo il modo di far "Parlare" HA sugli altoparlanti collegati direttamente al RaspBerry, alltrimenti è tutto pronto per la sostituzione.
